pyproject.toml

```diff
@@ -38,22 +40,21 @@ classifiers = [
 dynamic = [
   "version",
 ]
-[project.optional-dependencies]
-docs = [
+optional-dependencies.docs = [
```

Comment:
pyproject-fmt version 2 has a fairly big change here, reformatting these from "expanded" to "inline".

I slightly prefer the old version, but am not too bothered really. 

There's a chance pyproject-fmt might add an option to retain the old version. We could revert these changes for now, see if it's added by the next monthly autoupdate.

What do others think?

Reply:
I agree that the old version is neater.

If this is just how things are with the latest version, that's fine.
If you think we're just going to revert this change in a month though, I'd rather keep it as it is for now than confusing the blame for this file.